hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -1156,10 +1202,10 @@ object LoadVCF {
       .toMap
 
     val infoHeader = header.getInfoHeaderLines
-    val (infoSignature, infoAttrs, infoFlagFields) = headerSignature(infoHeader, callFields, entryFloatType)
+    val (infoSignature, infoAttrs, infoFlagFields) = headerSignature(infoHeader, callFields, floatType)
```

Comment:
this should be TFloat64(), I think

Reply:
OK, I fixed that and simplified the other change in `headerField`.